### ISSUE TITLE
Use last focused container for external links

### ISSUE THREAD

**Issue by andrejohansson** [mozilla/multi-account-containers#2306]
This is an addition/variation to #706.

# Problem description

Clicking links from external programs (slack, cli´s that do oauth authentication etc) often breaks the flow by opening the link in a default container. 

# Other solutions

- To always ask which container on external link: This seems to be a bit invasive (at least for me).
- Assign sites to container: This doesnt always work, I actually want different containers even for some sso accounts (i.e. github private and github work is the same account but depending on my work I like to open them in different containers).

# Additional information

I usually wok in "roles" so to speak...I'm at work, I'm at home, I'm working for my clients etc. These roles tend to use their own containers or a set of containers that are related.

# Proposed solution
I would like to get a flow going where I don´t get interrupted but can still cover more cases, so I suggest an option where opening a link from an external program would open a tab in **the last focused tab in the last focused window**. 

This way, I wouldn´t be bothered with questions. Oauth flows would work as long as I'm I have focus in the correct container.

Example:
I have a tab open in my "work" container. Then i click a link in slack to another app in our sso ecosystem. Firefox automatically opens this in the work container since thats the last focused, even if the url to is not assigned in any way and I get now questions about containers. The resulting experience for me is that I get signed in automatically in the correct container to a new app without any hassle.

**Comment by achernyakevich-sc** [comment]
I'm not sure that feature you propose will be implemented in scope of MAC add-on as it looks like Firefox itself does not provide suitable API. This is a bad news. But... There is a good news for you that you could get resolved your problem with `work` container the different way. :)

I would propose you to try Simple Tab Groups add-on that is greatly integrated with MAC add-on. All that you need is create group of tabs `work` and set `work` container as sticky to it. So if `work` groups is open in the browser then any link that should be open in Firefox will be open in the `work` container as it is sticky to the `work` group of tabs. The same way you could create `home` group and make it sticky to `home` container. But you are not limited and you still could have `whatever` group that will have not any sticky container and this way in `whatever` group external links will be open in the default container.

If this will help you then probably it would be reasonable to close this issue and this way to show other that it really works. :)

**Comment by andrejohansson** [comment]
@achernyakevich-sc this is interesting, I will test this for a while and report back!

**Comment by andrejohansson** [comment]
That worked really well, thank you @achernyakevich-sc for the tip!

**Comment by nekromoff** [comment]
Yes, please, otherwise some external login (OAuth flows) fail as they keep opening on the default, not desired container!
